src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -186,7 +186,7 @@ public void updateFilteredAssignmentList(Person person) {
         }
 
         @Override
-        public List<Assignment> getFilteredAssignmentList(Name name) {
+        public List<Assignment> getPersonAssignmentList(Name name) {
```

Comment:
If you decide to change the name back to ```getFilteredAssignmentList```, this name may need to change back as well